mysql-test/suite/rocksdb/t/ddse_upgrade_row_lock.test

```diff
@@ -0,0 +1,33 @@
+--source include/have_rocksdb.inc
+--source include/have_innodb_ddse.inc
+
+
+--let $MYSQLD_DATADIR= `select @@datadir`
+
+--let $shutdown_server_timeout= 10
+--source include/shutdown_mysqld.inc
+
+# save data
+let $tmp_dir = $MYSQLTEST_VARDIR/tmp;
+let $saved_datadir = $tmp_dir/saved_datadir;
+force-cpdir $MYSQLD_DATADIR $saved_datadir;
+
+# There are 3 place during DD upgrade which may involves a lot row locks
+# - insert select statement: copy data from one storage engine to another storage engine, this only happens for DD SE upgrade, with this change, no row lock is hold
```

Comment:
The lines appear to be too long, wrap around column 80?

Reply:
Fixed..